### DISCUSSION TITLE
Frigate - configure alerts time (example detect only on Monday from 8PM to 6AM)

### DISCUSSION THREAD

**Discussion by karopodgajecki** [blakeblackshear/frigate#14890] [General]
Hello, 

Im using frigate (I also added integration to HA), but I have question I didn't find anything in Frigate docs about supressing alerts (detection) based on day / time I'm missing something ?

I know that in Home assistant blueprint you can configure hours of alerts - but you cannot configure days of week when they should be enabled (from what I know)

**Comment by hawkeye217** [comment]
No, this is not currently possible with Frigate directly.

You can enable/disable detection for specific cameras with a Home Assistant automation, but this may not be exactly what you're looking for.

Feel free to leave a thumbs up on this open feature request: https://github.com/blakeblackshear/frigate/issues/13038

**Reply by karopodgajecki** [reply]
Thanks for replay - closing this one :)
